shardingsphere-jdbc/shardingsphere-jdbc-core/src/test/java/org/apache/shardingsphere/driver/jdbc/core/statement/ShardingSphereStatementTest.java

```diff
@@ -58,7 +58,6 @@ public void assertGetGeneratedKeys() throws SQLException {
             generatedKeysResultSet = statement.getGeneratedKeys();
             assertTrue(generatedKeysResultSet.next());
             assertThat(generatedKeysResultSet.getLong(1), is(6L));
-            assertFalse(statement.execute(String.format(sql, 1, 1, "init"), new String[]{"no"}));
```

Comment:
Refer to https://github.com/apache/shardingsphere/pull/19878#issuecomment-1207174040.